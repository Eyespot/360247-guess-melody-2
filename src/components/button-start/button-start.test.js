import React from 'react';
import renderer from 'react-test-renderer';
import ButtonStart from "./button-start.jsx";

it(`ButtonStart component renders correctly after relaunch`, () => {
  const tree = renderer
    .create(<ButtonStart
      onClick={()=>{}}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
