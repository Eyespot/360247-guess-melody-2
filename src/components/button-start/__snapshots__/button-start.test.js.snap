// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonStart component renders correctly after relaunch 1`] = `
<button
  className="welcome__button"
>
  <span
    className="visually-hidden"
  >
    Начать игру
  </span>
</button>
`;
